oci/private/image.sh.tpl

```diff
@@ -98,6 +98,17 @@ for ARG in "$@"; do
         (--env=*\${*}* | --env=*\$*) ENV_EXPANSIONS+=(${ARG#--env=}) ;;
         (--output=*) OUTPUT="${ARG#--output=}" ;;
         (--workdir=*) WORKDIR="${ARG#--workdir=}" ;;
+        (--labels-file=*)
+          while LANG=C IFS= read -r in || [ -n "$in" ]; do
```

Comment:
it forces some programs to understand this to be the locale, rather than some other default they might choose, which in turn can cause non-utf8 encodings. I don't think we actually need this in practice, since we are not calling any programs in here. will remove.

Reply:
gotcha. we can always add it later when there's an issue that needs this.